bot/src/main/java/com/community/tools/dto/OpenAiRequestDto.java

```diff
@@ -0,0 +1,40 @@
+package com.community.tools.dto;
+
+import java.util.Collections;
+import java.util.List;
+
+public class OpenAiRequestDto {
+  private String model;
+  private List<Message> messages;
+
+  public OpenAiRequestDto(String model, String prompt) {
+    this.model = model;
+    this.messages = Collections.singletonList(new OpenAiRequestDto.Message("user", prompt));
+  }
+
+  public String getModel() {
```

Comment:
I think better use lombok instead getters and setters

Reply:
had a problem with it because of inner class but will try)